src/test/java/org/openrewrite/java/migrate/javax/RemoveEmbeddableIdTest.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2024 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.javax;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class RemoveEmbeddableIdTest implements RewriteTest {
```

Comment:
It would be good to include the following scenarios:

- `Class1` annotated with `@Entity` and defines a field of type `Class2` annotated with `@Embedded`. Then `Class2`'s definition is annotated with `@Embeddable` and defines a field of any type annotated with `@Id`
- `@Embeddable ` is used on a sub class where this subclass uses `@Id`

Reply:
Does this recipe also have to handle the `@Embedded` case? The rule only mentioned `@EmbeddableId`

Reply:
In the second case, is `Class1` defining `Class2` or `Class2SubClass` with `@EmbeddableId`?
And should the recipe be expected to run?